testkit/src/main/java/org/apache/calcite/test/SqlOperatorTest.java

```diff
@@ -7909,6 +7909,45 @@ private static void checkArrayConcatAggFuncFails(SqlOperatorFixture t) {
         "17357000", "BIGINT NOT NULL");
   }
 
```

Comment:
only has the definition of function, where is the implementation of the function

Reply:
I will add a Quidem test https://github.com/apache/calcite/pull/3231#issuecomment-1569341799